examples/py/async-binance-cancel-option-order.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
@Dan-krm thanks for creating these examples, can you please create one additional file with some public calls? (orderbook, ticker, etc)

Reply:
Yeah I'll add some more public examples